easybuild/framework/easyblock.py

```diff
@@ -1068,8 +1067,9 @@ def make_module_footer(self):
         footer = [self.module_generator.comment("Built with EasyBuild version %s" % VERBOSE_VERSION)]
 
         # add extra stuff for extensions (if any)
-        if self.cfg['exts_list']:
-            footer.append(self.make_module_extra_extensions())
+        extensions = self.cfg['exts_list']
+        if extensions:
```

Comment:
it should be fine to just drop this condition imho... with `self.exts_all` we had to be more careful since the default value was `None`, but `self.cfg['exts_list']` is an empty list by default so it's fine to always iterate over it